docs/app/services/plunker/templates/main_ts.txt

```diff
@@ -1,11 +1,11 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { CheckboxModule, ToggleSwitchModule } from 'ux-aspects';
 import { AppComponent } from './app';
+${imports}
 
 @NgModule({
-    imports: [BrowserModule, CheckboxModule, ToggleSwitchModule],
+    imports: [BrowserModule, ${modules}],
```

Comment:
If there are no modules imported then we are going to be left with `[BrowserModule, ]`. The comma should only appear when we add modules as IE can sometimes mistake this for an extra array element